projects/core/src/lib/components/toc/toc-node-details/layer-transparency/layer-transparency.component.ts

```diff
@@ -43,10 +47,12 @@ export class LayerTransparencyComponent {
   }
 
   public resetOpacity() {
-    if (!this.layerId) {
-      return;
-    }
-    this.store$.dispatch(setLayerOpacity({ layerId: this.layerId, opacity: undefined }));
-  }
+    this.layer$.pipe(take(1)).subscribe(layer => {
```

Comment:
If you replace `this.layer$` with `this.store$.select(selectLayer(this.layerId))` you can save a public property and initialization.